testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -26,10 +26,12 @@ Feature: Disable and re-enable monitoring of the server
     And I should see a list item with text "Tomcat (Java JMX)" and a failing bullet
     And I should see a "Restarting Tomcat and Taskomatic is needed for the configuration changes to take effect." text
     And file "/etc/rhn/rhn.conf" should contain "prometheus_monitoring_enabled = 0" on server
-    And file "/etc/sysconfig/tomcat" should not contain "Dcom.sun.management.jmxremote.port=3333 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
-    And file "/etc/rhn/taskomatic.conf" should not contain "Dcom.sun.management.jmxremote.port=3334 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
+    And port "3333" should be closed
+    And port "3334" should be closed
     And file "/usr/lib/systemd/system/tomcat.service.d/jmx.conf" should not exist on server
     And file "/usr/lib/systemd/system/taskomatic.service.d/jmx.conf" should not exist on server
+    And port "5556" should be closed
+    And port "5557" should be closed
```

Comment:
Sorry, I overlooked this before actually: all these ports are effectively opened / closed only *after* the restart, so we should move the port checks down to the respective scenarios after each restart (`Check that monitoring is disabled / enabled using the UI`, maybe just `using the UI` is no longer true then). `3333` and `3334` should be closed in both cases, but more importantly `5556` and `5557` should be opened when monitoring is enabled and closed when monitoring is disabled.

Reply:
Makes sense. I will adjust this after lunch. Thank you for having another look, Johannes!

Reply:
Fixed with 8ea5645375f61beb68c58714a6443e50b63f68b0.